x-pack/metricbeat/module/gcp/constants.go

```diff
@@ -16,11 +16,12 @@ const (
 
 // Metricsets / GCP services names
 const (
-	ServiceCompute       = "compute"
-	ServicePubsub        = "pubsub"
-	ServiceLoadBalancing = "loadbalancing"
-	ServiceFirestore     = "firestore"
-	ServiceStorage       = "storage"
+	ServiceCompute        = "compute"
+	ServicePubsub         = "pubsub"
+	ServiceLoadBalancing  = "loadbalancing"
+	ServiceCloudFunctions = "cloudfunctions"
```

Comment:
Do we want to make this list in alphabetical order? This is not a required change of course!

Reply:
Ok, CI was also failing so I took advantage to sort this and re-trigger CI with the pushed commit